widgets/viewports/viewportbase.h

```diff
@@ -180,6 +180,7 @@ class ViewportBase : public QOpenGLWidget, protected QOpenGLFunctions_1_4 { // g
     void renderSphere(const Coordinate &pos, float radius, const QColor &color, const RenderOptions & options = RenderOptions());
     void renderCylinder(const Coordinate &base, float baseRadius, const Coordinate &top, float topRadius, const QColor &color, const RenderOptions & options = RenderOptions());
     void renderSkeleton(const RenderOptions & options = RenderOptions());
+    void genVertBuffer(const RenderOptions &options = RenderOptions());
```

Comment:
and actually, i’d go for a non abbreviated name, like `generateSkeletonGeometry`